examples/multisig/lib.rs

```diff
@@ -135,6 +138,8 @@ mod multisig {
         pub transferred_value: Balance,
         /// Gas limit for the execution of the call.
         pub gas_limit: u64,
+        /// The call flag allowing reentry
```

Comment:
```suggestion
        /// If set to true the transaction will be allowed to re-enter the multisig contract.
```

Reply:
well, not necessarily multisig contract - it will apply to the contract we call which is specified by `t.callee`

Reply:
But multisig is always the caller and hence the one who blocks reenter.